### DISCUSSION TITLE
Can I add a modified example?

### DISCUSSION THREAD

**Discussion by ahmedshahriar** [mesa/mesa#1279] [Q&A] [answered]
So I recently did a sample project where I modified the `boid_flockers` example. Can I submit a PR with it?

1. I added a property color to class Boid, and initialized every boid with color=1, except for one boid, which will have color=2. Modified boid_draw(agent) method of server.py to draw in "Blue" if agent.color equals to 1, and "Red" if it is 2. ( all but one boids blue, and one red)
2. I modified boid's step() method the following way. If boid's color is 2 (red), get a list of all boids within the radius of 2 of that boid. All the boids in that list that are currently blue, should turn red (changed their color property to 2). As a result, as the model runs and red boids "bump" into blue ones, more and more boids will turn red.
3. I added a graph visualization to the program so that it shows the graph of the number of red-colored boids under the main "window", as the model runs. 
4. I also modified the program so that each boid stays red only for 25 ticks (steps), and after that, it turns into a new color: green. Green boids cannot change color anymore.
5. Finally, I modified the graph underneath so that it shows simultaneously the number of red and the number of green boids. 

Final result - 

![boids_green](https://user-images.githubusercontent.com/40615350/163100917-e778f309-9067-4a31-975d-b345c259f14c.png)

![boids_red](https://user-images.githubusercontent.com/40615350/163100923-a0e73bb3-3ede-4f39-8c51-fb21b46ccce7.png)

**Comment by rht** [comment, marked as answer]
Hmm, how would you summarize the nature of your change in a few phrases or a sentence? While it makes sense to e.g. extend the virus on a network to use a directed graph (see #1223), and so showcasing a new separate feature ( i.e. directed graph), I can't pinpoint such feature from your modification.

For now, I think you should park the model in a repo under your GH account (please do share the link!). Also, check out the discussion in https://github.com/projectmesa/mesa/discussions/1158.

**Reply by ahmedshahriar** [reply]
Thanks for your response. I think I will extend my project and create a separate discussion thread under [Show and tell](https://github.com/projectmesa/mesa/discussions/categories/show-and-tell) and share my GH repo there.

Btw, in my case, it's more like a `zombie bird` xD

Thanks again!
